spec/features/availability_spec.rb

```diff
@@ -152,18 +152,33 @@
       allow(Rails.configuration).to receive(:use_alma).and_return(true)
     end
 
-    describe 'Electronic Holdings', js: true do
-      it 'within the online section it does not display links', unless: in_ci? do
-        stub_holding_locations
+    describe 'Electronic Holdings' do
+      it 'within the online section, it displays electronic portfolio links' do
         visit '/catalog/99122306151806421'
-        expect(page).not_to have_selector '.availability--online a'
+        expect(page).to have_text '1869 - 1923: Biodiversity Heritage Library Free'
+        expect(page).to have_text 'Available from 1869 volume: 1 issue: 1.'
+
+        # Electronic portfolio link with an embargo
+        expect(page).to have_text '1990 - 2020: ProQuest Central'
+
```

Comment:
This is ok based on the published fixture in bibdata 
`<datafield tag="954" ind1="" ind2="">
    <subfield code="a">53443322630006421</subfield>
    <subfield code="b">&gt;=</subfield>
    <subfield code="c">1</subfield>
  </datafield>`

I will approve this PR based on the display. 
We need though to improve the calculation for the emabrgo date. I opened issue https://github.com/pulibrary/bibdata/issues/1147 with more information on the embargo date. 
Thank you !